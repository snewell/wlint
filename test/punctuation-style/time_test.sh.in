10:00am
12:00 pm
2:00 AM
3:00 P.M.
4:00 aM
5:00 Pm
6:00 a.M.
