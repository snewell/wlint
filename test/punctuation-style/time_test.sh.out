<stdin>-1:0 time.missing-periods
<stdin>-1:0 time.missing-space
<stdin>-2:0 time.missing-periods
<stdin>-3:0 time.missing-periods
<stdin>-3:0 time.uppercase-APM
<stdin>-4:0 time.uppercase-APM
<stdin>-5:0 time.missing-periods
<stdin>-5:0 time.uppercase-apM
<stdin>-6:0 time.missing-periods
<stdin>-6:0 time.uppercase-APm
<stdin>-7:0 time.uppercase-apM
